ang/crmUi.js

```diff
@@ -1002,16 +1002,22 @@
         scope: {
           crmDocumentTitle: '='
         },
+        limit: 'ae',
         link: function(scope, $el, attrs) {
           function update() {
             $timeout(function() {
               var newPageTitle = _.trim($el.html()),
-                newDocumentTitle = scope.crmDocumentTitle || $el.text();
+                newDocumentTitle = scope.crmDocumentTitle || $el.text(),
+                cls = $el.attr('class').split(' '),
+                classes = _.filter(cls, function(c) {
+                  return c.indexOf('ng-') !== 0;
```

Comment:
Well `crm-page-title` is the class I think we should add no matter what. I started to leave the PR at just that, then got inspired to add other stuff from the `class` attribute, then realized there was some `ng-*` junk in there so stripped it out.

I don't feel strongly one way or the other about your suggestion. I'm tempted to just take it back to the simpler `.addClass('crm-page-title')` and be done with it, since that satisfies the use case.